r/tests/testthat/test-parquet.R

```diff
@@ -197,6 +197,8 @@ test_that("Maps are preserved when writing/reading from Parquet", {
 })
 
 test_that("read_parquet() and write_parquet() accept connection objects", {
+  skip_if_not_available("snappy")
```

Comment:
Yeah, I've been thinking about how may of these we need + it would be great if we actually tested this with our default and then fell back on sanitizers only. Either compression should be totally fine, but it would be nice to have coverage on both. But we should do that generally. I'll make a Jira for that + we can do that outside of the release pressure.

Reply:
https://issues.apache.org/jira/browse/ARROW-16385